pkg/collector/pipeline.go

```diff
@@ -48,7 +48,7 @@ type ReportBatch struct {
 
 	// An arbitrary set of extra data that you can attach to this batch of
 	// reports.
-	Annotation interface{}
+	Annotations
```

Comment:
Is this shorthand for defining a field whose type and name are the same?

Reply:
That, and it also means that the methods of the field can be called directly on the containing struct.  So you can say

``` go
var batch ReportBatch
batch.SetAnnotation("foo", "bar")
```

instead of needing

``` go
var batch ReportBatch
batch.Annotations.SetAnnotation("foo", "bar")
```